tests/e2e/logging/test_openshift-logging.py

```diff
@@ -39,49 +45,53 @@ def finalizer():
     assert ocp_logging_obj.set_rbac(
         yaml_file=constants.EO_RBAC_YAML, resource_name='prometheus-k8s'
     )
-    assert ocp_logging_obj.create_elasticsearch_subscription(constants.EO_SUB_YAML)
+    logging_version = config.ENV_DATA['logging_version']
+    subscription_yaml = templating.load_yaml(constants.EO_SUB_YAML)
+    subscription_yaml['spec']['channel'] = logging_version
+    helpers.create_resource(**subscription_yaml)
+    assert ocp_logging_obj.get_elasticsearch_subscription()
 
-    # Deploys cluster-logging operator on the project openshift-logging
+    # Deploys cluster-logging operator on the project openshift-loggingno nee
     ocp_logging_obj.create_namespace(yaml_file=constants.CL_NAMESPACE_YAML)
     assert ocp_logging_obj.create_clusterlogging_operator_group(
         yaml_file=constants.CL_OG_YAML
     )
-    assert ocp_logging_obj.create_clusterlogging_subscription(
-        yaml_file=constants.CL_SUB_YAML
+    cl_subscription = templating.load_yaml(constants.CL_SUB_YAML)
+    cl_subscription['spec']['channel'] = logging_version
+    helpers.create_resource(**cl_subscription)
+    assert ocp_logging_obj.get_clusterlogging_subscription()
+    cluster_logging_operator = OCP(
+        kind=constants.POD, namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
     )
+    logger.info(f"The cluster-logging-operator {cluster_logging_operator.get()}")
 
-    # Creates storage class
-    cbp_obj = helpers.create_ceph_block_pool()
-    sc_obj = helpers.create_storage_class(
-        interface_type=constants.CEPHBLOCKPOOL,
-        interface_name=cbp_obj.name,
-        secret_name=constants.DEFAULT_SECRET,
-        reclaim_policy="Delete"
-    )
-    assert ocp_logging_obj.create_instance_in_clusterlogging(sc_name=sc_obj.name)
+    create_instance()
+
+
+@retry(CommandFailed, 10, 10, 3)
```

Comment:
could you please the parameter names?
retry=10, etc.

Reply:
Done